src/cli.c

```diff
@@ -245,9 +245,17 @@ static int flatten_sized_text(quicly_sendbuf_vec_t *vec, void *dst, size_t off,
         "world\nhello world\nhello world\nhello world\nhello world\nhello world\nhello world\nhello world\nhello world\nhello "
         "world\nhello world\nhello world\nhello world\nhello world\nhello world\n";
 
-    const char *src = pattern + off % 12;
-    assert(src + len - pattern <= sizeof(pattern) - 1); /* pattern is bigger than MTU size */
-    memcpy(dst, src, len);
+    while (len != 0) {
```

Comment:
do we really need a pattern, or can we simply replace this with a single letter, say 'a'?

Reply:
We use the repetition of "hello world\n" as the expected pattern in our end-to-end tests (see t/e2e.t). It is a way of trying to detect transmission of corrupt data. I'm not sure how effective those tests are, but I do not think there is a need to change the approach at this moment.